pkg/storage/secret/migrator/migrator.go

```diff
@@ -152,4 +152,26 @@ func (*SecretDB) AddMigration(mg *migrator.Migrator) {
 		Cols: []string{"namespace", "label", "active"},
 		Type: migrator.IndexType,
 	}))
+
+	// Owner Reference columns
+	mg.AddMigration("add owner_reference_api_version column to "+TableNameSecureValue, migrator.NewAddColumnMigration(secureValueTable, &migrator.Column{
```

Comment:
🤔  it may be more useful to split this into two field `api_group` and `api_version`

annoying, but the "apiVersion" property is really {group}/{version} -- the version can change and we are still talking about the same thing